tests/test_api_client.py

```diff
@@ -72,6 +72,7 @@ def test_init_successful(self, mock_requests_get):
                 "X-API-KEY": os.environ["VM_API_KEY"],
                 "X-API-SECRET": os.environ["VM_API_SECRET"],
                 "X-PROJECT-CUID": os.environ["VM_API_MODEL"],
+                "X-MONITORING": "False",
```

Comment:
I think it should be fine to use a boolean. The HTTP client will convert it to a string for us. We just need to ensure we parse the incoming values properly on the backend side.


```suggestion
                "X-MONITORING": False,
```

Reply:
HTTP client doesn't covert the header into string if we pass it boolean.
```
InvalidHeader: Header part (True) from ('X-MONITORING', True) must be of type str or bytes, not <class 'bool'>
```
solution:
we can pass `vm.init(monitoring=True)` then covert in to str when passing to http client.